jinahub/encoders/audio/VGGISHAudioEncoder/vggish_audio_encoder.py

```diff
@@ -33,6 +35,9 @@ def __init__(self,
         self.pca_path = pca_path
         self.default_traversal_paths = default_traversal_paths or ['r']
 
+        if not os.path.exists(self.model_path) or not os.path.exists(self.pca_path):
+            subprocess.call('scripts/download_model.sh')
+
```

Comment:
please avoid to use `subprocess`, use `requests` to download the model instead.

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/31/commits/adc062852447ab0fa23e4b5dbc3cbcdba9e083ff) for the change.